src/RenderBoard.ts

```diff
@@ -33,6 +20,30 @@ export function renderBoard(board: Board): String {
     return result;
 }
 
-export function render(graphicalBoard: String) {
+function marbleValuetoANSIColorCode(marble: number): String {
+    let marbleColor: String;
+    const redMarble = `\u001b[31m \u2022 \u001b[0m`;
+    const blueMarble = `\u001b[34m \u2022 \u001b[0m`;
+    const whiteMarble = `\u001b[37m \u2022 \u001b[0m`;
+    const emptyMarble = `   `;
```

Comment:
You can probably extract those from the function and export them to reuse in tests